eva-lib/src/main/java/uk/ac/ebi/eva/lib/configuration/EvaProperties.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2016-2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.lib.configuration;
+
+import org.hibernate.validator.constraints.NotBlank;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+import org.springframework.validation.annotation.*;
+
+@ConfigurationProperties(ignoreUnknownFields = false, prefix = "eva")
+@Component
+public class EvaProperties {
+
+    private String version;
+
+    @javax.validation.constraints.NotNull
+    private Collections collections;
+
+    public static class Collections {
+        @NotBlank
```

Comment:
What about using `javax.validation.constraints.Size` with a minimum size of 1? That would make unnecessary to import a Hibernate package.